Performance Enhancements for Sparse Embedding Lookups (#57956)
tensorflow/tensorflow: RELEASE.md

        `tf.keras.metrics.F1Score`, and `tf.keras.metrics.R2Score`.
    *   Added activation function `tf.keras.activations.mish`.

> *   Embeddings:

Reviewer: ```suggestion
*   `tf.nn`
```

>     *   `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`
>         now support ids and weights described by RaggedTensors, which are more
>         memory efficient than SparseTensors. The new boolean argument
>         `allow_fast_lookup` enables a simplified and typically faster lookup
>         procedure.

Reviewer: ```suggestion
    *   `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`
        now support ids and weights described by `tf.RaggedTensor`.
    *   Added a new boolean argument `allow_fast_lookup` to 
        `tf.nn.embedding_lookup_sparse` and `tf.nn.safe_embedding_lookup_sparse`.
```

Author: A brief explanation of the purpose of `allow_fast_lookup` might be beneficial.

Reviewer: Feel free to add it back.
